database/migration/src/main/resources/metadata_changesets/partition_metadata_table.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog objectQuotingStrategy="QUOTE_ALL_OBJECTS"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet id="update_indexes_metadata_table" author="cahrens" dbms="mysql,postgresql,hsqldb,mariadb">
```

Comment:
TOL/nit: I would be tempted to update the indexes in a separate xml file to the compression/partitioning, just to make it easier to find them by name later on (eg scanning through the changelog this file is called `partition_metadata_table.xml` but is also currently doing the index addition.

It might also make it easier to associate together the changes related to the partitioning and to the indexes.

Reply:
Since the change in indexes is driven by the primary key change, I think we want this to be in the same xml file as where the primary key change occurs. And the primary key change is being done to partition the table, so I feel OK keeping the name as-is. But if folks think it would be good to include "primary key" in the xml filename, I can try to come up with a more descriptive name.

Reply:
I think changing the primary key is the single most significant change in this file so +1 to Chris